packages/calcite-components/src/components/panel/panel.tsx

```diff
@@ -718,7 +718,7 @@ export class Panel
     );
 
     return (
-      <Host onKeyDown={panelKeyDownHandler}>
+      <Host onKeyDown={this.panelKeyDownHandler}>
```

Comment:
noop change, but this makes it easy for the codemod to refactor-out `<Host>`

---

I earlier mentioned that special utils would be needed to replace `<Host>` in Lit, but after reading `<Host>` source code in Stencil, that doesn't seem to be necessary